.circleci/config.yml

```diff
@@ -160,6 +176,18 @@ jobs:
       - run: |
           [[ $CIRCLE_TAG == *-* ]] || TELEPRESENCE_VERSION=$CIRCLE_TAG make promote-to-stable
 
+  "release-chart":
+    executor: vm-linux
+    steps:
+    - checkout
+    - install-helm
+    - run: make lint
```

Comment:
Why have a `make lint` here? We don't do any other checks in the release jobs.

Reply:
I was being too defensive since I wanted to ensure the chart linted prior to pushing, but since we ensure lint on merge to master, then there's really no _real_ reason to lint again. So I'll remove it.